easybuild/easyconfigs/a/ANTLR/ANTLR-2.7.7-CrayGNU-17.08-Python-2.7.13.eb

```diff
@@ -0,0 +1,34 @@
+# contributed by Luca Marsella (CSCS)
+easyblock = 'ConfigureMake'
+
+name = 'ANTLR'
+version = "2.7.7"
+versionsuffix = '-Python-%(pyver)s'
```

Comment:
use `cray-python/17.06.1` & `versionsuffix='-python2'`

Reply:
rename 
`ANTLR-2.7.7-CrayGNU-17.08-Python-2.7.13.eb`
into
`ANTLR-2.7.7-CrayGNU-17.08-python2.eb`

Reply:
what's the difference with `https://github.com/antlr/antlr4/releases`

Reply:
https://github.com/eth-cscs/production/blob/master/easybuild/easyconfigs/p/PyExtensions/PyExtensions-2.7-CrayGNU-17.08.eb

Reply:
First two comments: done.
I don't know what are the differences, between ANTLR2 and ANTLR4. As agreed, I will continue with the ANTLR2 for the moment.

Reply:
`ANTLR4` changed completely with respect to `ANTLR2` and it would require a major revision of `NCO` build: since this is just a dependency, I think keeping `ANTLR` is a good decision.

Reply:
OK, thanks Luca

Reply:
you still need to rename this .eb

Reply:
I have already renamed the file; so I remove therefore the old file: easybuild/easyconfigs/a/ANTLR/ANTLR-2.7.7-CrayGNU-17.08-Python-2.7.13.eb